### ISSUE TITLE
Cannot load database.yml

### ISSUE THREAD

**Issue by sagenthings** [filoghost/HolographicDisplays#255]
I am getting this message. I imported my database.yml into the folder after modifying it and i'm getting this back when i run the server.

[Server] Server thread/ERROR Cannot load plugins/HolographicDisplays/database.yml

Can anyone please help me?

**Comment by Federe76** [comment]
I add to this. I had several HD's set. I made them from inside the game using commands. After I figured out that I could edit the file and reupload it I did exactly that. But now I have no HD's.

After I uploaded the file:
- All HD's displayed correctly.
- I ran /hd reload.
- All HD's dissapeared.
- Tried restarting the server and nothing.
The console shows the following (extended version on .txt):

Cannot load plugins/HolographicDisplays/database.yml
org.bukkit.configuration.InvalidConfigurationException: while parsing a block mapping
in 'string', line 13, column 3:
location: world, 320.147, 93.000 ...
^
expected , but found ''
in 'string', line 93, column 6:
- 'icon: golden_shovel'
^

[Error Holographic Displays.txt]
(https://github.com/filoghost/HolographicDisplays/files/4359975/Error.Holographic.Displays.txt)
